### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by renatocarvalho**
Convert this guide to from portuguese to english.

cc/ @rafaelrtp @flavioludgero @romulomachado @sales @vhaidamus

### CODE REVIEW

**Comment by romulomachado** [comment]
@startae/startae-team finished updating. Review.

File: README.md [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
 
-Assista [MUNDO DOS NEGÓCIOS](https://youtu.be/cPbl26Fw-dk).
+Have you ever heard someone saying "POOOORRA IVAN"? Or have you ever been called Ivan, even though that's not your name? And after that, have you ever looked around the office for any Ivan and could not found anyone with that name?
+
+Find more about Ivan [here](https://youtu.be/cPbl26Fw-dk).
 
 ## Oh no!
 
-TODO
+Once upon a time, Rafael and Victoria were starting a new project. They were supposed to redesign the experience for a coaching platform and were watching screencasts to better understand how the system worked. As the lady explained how to add and edit participants little did she notice that the edit button was right next to the delete trash bin. She then accidentally clicked the delete icon and, since there was no alert, deleted a real participant from the platform. Losing her composure and yelling "OOOOH NO!!!"
```
Comment (renatocarvalho) [inline README.md]: I couldn't understand this `...edit participants little...`

File: README.md [hunk 2]
```diff
@@ -1,35 +1,43 @@
...
+You probably want to say it, don't you?
+You have probably already have said it, didn't you?
+
+And you have probably heard someone singing ["Beautiful"](https://www.youtube.com/watch?v=_FE194VN6c4) in a highly pitched voice. We actually don't know how this started, but that will happen EVERY TIME you say that word.
 
 ## WTFriday
 
-Você entrou no Slack e se deparou os gifs mais loucos do mundo? Provavelmente
- estamos na wtfriday: dia de acordar, tomar aquele café e contribuir com o seu crazy gif.
+You went to Slack and found the weirdest gifs ever? Probably, it's wtfriday: you wake up, drink your coffe and contribute with the craziest gif you can find.
```
Comment (renatocarvalho) [inline README.md]: perhaps "You opened Slack and found..."

**Comment by renatocarvalho** [comment]
Everything else looks awesome :dancers:

File: README.md (hunk 1 above)
Comment (vhaidamus) [inline README.md]: vou melhorar esse texto ai ;)

Comment (romulomachado) [inline README.md]: @vhaidamus en, please.

**Comment by renatocarvalho** [comment]
@vhaidamus @romulomachado :shipit:

**Pull request merged**